presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -219,6 +231,25 @@ std::unique_ptr<TaskInfo> TaskManager::createOrUpdateErrorTask(
   return ss.str();
 }
 
+void TaskManager::getDataForResultRequests(
```

Comment:
Nit : Just getData(...) should work also. ResultRequests is redundant.

Reply:
There is already a getData() method. Will keep this as for readability.